topo/dva/devices_block-yellow.yaml

```diff
@@ -7,17 +7,10 @@ device_mac_placements:
         switch: "nz-kiwi-t2sw2"
         port: 1
         connected: True
-    "9a:02:57:1e:8f:03":
-        switch: "nz-kiwi-t2sw3"
-        port: 1
-        connected: True
 device_mac_behaviors:
     "9a:02:57:1e:8f:01":
         segment: INFRA
         role: "red"
     "9a:02:57:1e:8f:02":
         segment: BOS
         role: "green"
-    "9a:02:57:1e:8f:03":
-        segment: BOS
-        role: "yellow"
```

Comment:
I don't think this file needs to be updated since it is only used in test_topology.

Reply:
I would want clarification as to why this file is changing b/c of radius testing.  My impression was that radius integration should be behavior neutral, the only difference is that the result comes from radius rather than a separate file.    If it's an adjacent cleanup of the test for some reason (e.g. it was wrong before), then please do that as a separate CL so it's crystal clear as to how all changes relate to radius integraiton tests.

Reply:
I shall revert this file.

Reply:
Done.